corehq/motech/value_source.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
+
+from __future__ import unicode_literals
+from couchdbkit.ext.django.schema import DocumentSchema
+
+
+def recurse_subclasses(cls):
```

Comment:
corehq/motech/openmrs/finders.py:42 is looking for this in its old location. That's what's breaking the build.